cli/src/merge_tools/external.rs

```diff
@@ -58,6 +58,11 @@ pub struct ExternalMergeTool {
     /// `$left`, `$right`, `$base`, and `$output` are replaced with
     /// paths to the corresponding files.
     pub merge_args: Vec<String>,
+    /// If false (default), any exit status from the merge tool other than 0
+    /// will cause `jj resolve` to fail. If true, then any status between 1 and
+    /// 127 can be used to indicate that the conflict was not fully resolved,
+    /// and the output file should contain conflict markers.
+    pub merge_tool_error_status_on_conflict: bool,
```

Comment:
The choice `1 to 127` sounds a bit arbitrary. Maybe better to define a set or range of expected exit codes per tool? I assume merge drivers wouldn't use full range of status codes from 1 to 127.

The Git doc says "When the driver crashes (e.g. killed by SEGV), it is expected to exit with non-zero status that are higher than 128." This implies that the driver is executed through a system shell (i.e. `system()` instead of `execv()`.) If the driver is directly executed, SEGV wouldn't be mapped to an exit status code.

Reply:
I replaced it with an array of exit codes to indicate a conflict. Do you think that works better?